### DISCUSSION TITLE
Welcome to GamecubeBT Discussions!

### DISCUSSION THREAD

**Discussion by sirjeannot** [sirjeannot/GamecubeBT#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dav0163** [comment]
Have you thought about adapting this for a gamecube controller?

**Reply by sirjeannot** [reply]
for this I would need a wavebird. unfortunately the wavebird is pure 2.4G non bluetooth. it would probably work with a nrf24l.
If I have some time and I find a cheap wavebird, why not :)
